### PR TITLE
chore: add github issue templates

### LINKED ISSUE #87: Add github issue templates for helm-backstage
### What needs to be done?
Add github issue templates in helm-backstage using the default templates in the janus-idp organization.

### PR DESCRIPTION
**Pull request by Zaperex**
## Description of the change

Add default github issue templates that match the organization defaults.

## Existing or Associated Issue(s)

Fixes #87 
## Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the `values.yaml` and added to the README.md. The [pre-commit](https://pre-commit.com/) utility can be used to generate the necessary content. Use `pre-commit run -a` to apply changes.
- [ ] JSON Schema template updated and re-generated the raw schema via `pre-commit` hook.
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+### Actual Behavior
+
+What actually happened? Please provide screenshots if applicable.
+
+### Additional information
+
+- Which devices are affected? ie Desktop, Table, Phone
+- Which browsers are affected? ie Chrome, Firefox, Safari, Edge
+- Which version are you using? (We only support the last 2 major versions.)
```
Comment (tumido) [inline .github/ISSUE_TEMPLATE/bug.md:29]: Helm doesn't support any version. In fact you can deploy any image with this chart. How is this relevant?

**Review by tumido — CHANGES_REQUESTED** [review]
I'm sorry @Zaperex , I feel like you're just copypasting something without actually caring about against which repository you're opening this.

These careless PRs waste reviewers time. Please make this relevant to the repository content.

File: .github/ISSUE_TEMPLATE/feature.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+name: Feature Request
+about: If you have a feature request
+labels: ['kind/feature', 'status/triage']
+---
```
[the remark this answers is not in the document]
Comment (Zaperex) [reply .github/ISSUE_TEMPLATE/feature.md:5]: enhancement is for an existing feature, while feature is for a new feature

**Comment by Zaperex** [comment]
Will backtrack on this in favor for more generic templates
